chia/wallet/did_wallet/did_wallet.py

```diff
@@ -227,10 +228,10 @@ async def create_new_did_wallet_from_coin_spend(
         inner_solution: Program = full_solution.rest().rest().first()
         recovery_list: List[bytes32] = []
         backup_required: int = num_verification.as_int()
-        if recovery_list_hash != Program.to([]).get_tree_hash():
+        if recovery_list_hash != NULL_TREEHASH:
```

Comment:
`git blame` says it's my fault that this is named `NULL_TREEHASH` rather than `NIL_TREEHASH`.

I wouldn't totally be against you renaming it `NIL_TREEHASH` in `chia.wallet.util.curry_and_treehash` and adding a `NULL_TREEHASH = NIL_TREEHASH  # deprecated, use NIL_TREEHASH` there (for now).

Totally optional.

Reply:
yeah, I went through and renamed it in `chia_rs`. I'll rename this one too.